src/main/java/pwr/sim/animal/Animal.java

```diff
@@ -53,7 +53,7 @@ public void move(int x, int y) {
         if(tile instanceof WaterTile) {
             return;
         }
-        nextPosition.move(newx, newy);
+        nextPosition.move(x, y);
```

Comment:
And this is fixing a leftover from when the method called in this line was `setPosition(newx, newy)`, correct?